review-please/action.yml

```
@@ -0,0 +1,22 @@
+name: review-please
+description: Add label when assigned reviewers
+
+inputs:
+
+  add-labels:
+    description: 'Labels list for pull request (comma-separated string), Defaults to ":pray: review"'
+    required: false
+    default: ":pray: review"
```

```suggestion
    default: "review"
```

デフォルトは絵文字無しで素直なラベルでいいんじゃないかな